src/main/java/org/assertj/vavr/internal/Maps.java

```diff
@@ -361,6 +387,10 @@ private static void assertNotNull(AssertionInfo info, Map<?, ?> actual) {
         return key -> !keys.contains(key);
     }
 
+    private static <K> Predicate<K> presentIn(Set<K> keys) {
+        return key -> keys.contains(key);
```

Comment:
Maybe `key` -> `e` or `elem`, `keys` -> `elements`. This predicate is more general.